lib/tasks/gettext.rake

```diff
@@ -90,4 +90,25 @@ namespace :gettext do
       end
     end
   end
+
+  task :extract_locale_names do
```

Comment:
No, `Vmdb::FastGettextHelper.register_locales` doesn't really need rails environment loaded.

It only uses `fast_gettext`, nothing else really.